lace/test_topology.py

```diff
@@ -198,3 +199,17 @@ def test_clean_segments(self):
         }
 
         self.assertEqual(cube.clean_segments(['random_segm', 'all']), ['all'])
+
+    def test_flip_faces(self):
+        raw_box = Mesh(vc('/unittest/serialization/obj/test_box_simple.obj'))
+        box = Mesh(v=raw_box.v, f=raw_box.f)
+        box.reset_normals()
+        original_vn = box.vn.copy()
+        original_f = box.f.copy()
+        box.flip_faces()
+        box.reset_normals()
+        self.assertEqual(box.f.shape, original_f.shape)
+        for face, orig_face in zip(box.f, original_f):
+            self.assertNotEqual(list(face), list(orig_face))
```

Comment:
Same comment as above. The conversion should be deterministic so we should be able to test for equality. I realize that there are multiple different ways that the same face can be defined, but our code should pick one. Maybe lowest vertex number should always be first?